_schemaV2.graphql

```diff
@@ -5441,6 +5466,18 @@ type Money {
   ): Float!
 }
 
+# A simple type to hold an amount and currency code
+input MoneyInput {
+  # Amount in the minor unit of currency
+  amountMinor: Int!
```

Comment:
This schema doesn't match the [v2 field definition of `MoneyInput`](https://github.com/artsy/metaphysics/blob/f885199a5ababedcd0b81fdbd55395dc48f9a98c/src/schema/v2/fields/money.ts#L170), which oddly isn't causing any v2 tests to fail, but should be looked at.